torch/nn/parallel/distributed.py

```diff
@@ -568,93 +560,29 @@ def __setstate__(self, state):
         self.__dict__.setdefault("require_forward_param_sync", True)
         self.__dict__.setdefault("require_backward_grad_sync", True)
         parameters, expect_sparse_gradient = self._build_params_for_reducer()
+        # Verify model equivalence.
```

Comment:
I think this doc should go before `dist._verify_model_across_ranks` a few lines above

Reply:
Thanks for catching this! Didn't realize that how this comment went to that line.

Corrected in #55877